### PR TITLE
Cinder implementation

### PR DESCRIPTION
**Pull request by giffels**
Hi folks,

thank you for the fast response to my last requests. However, I have a few more things I would like to add to your async OpenStack client.

1. Seems that Accept and Content Type headers are missing in the client. (see https://github.com/DreamLab/AsyncOpenStackClient/commit/34f648dbcc131756ce3d2c53ed5bdd5d589d21fc)

2. In case you are interested to support also the volumes API of Cinder, please have a look at https://github.com/DreamLab/AsyncOpenStackClient/commit/ac62956b8610d734c4e8de1eb529655429c2f74c.

I would really appreciate if you consider to merge this request as well. 

Thanks and best regards,
Manuel

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -65,6 +67,12 @@ Usage
     response = await nova.servers.create(server=specs)
     print(response)
 
+    volume = {"size": 200,
+              "imageRef": "image_id",
+              "name": "some_name"}
+
+    response = await cinder_client.volumes.create(volume=volume)
```
Comment (evemorgen) [inline README.rst]: Just to make this example more copy-paste friendly, I would fix this typo `cinder_client` -> `cinder`

File: src/asyncopenstackclient/client.py [hunk 2]
```diff
@@ -22,7 +22,10 @@ def __init__(self, api_name, resources, session=None, api_url=None):
         await self.get_credentials()
         self.api = API(
             api_root_url=self.api_url,
-            headers={'X-Auth-Token': self.session.token},
+            headers={'X-Auth-Token': self.session.token,
```
Comment (evemorgen) [inline src/asyncopenstackclient/client.py]: Wouldn't it be nice if it was `pep8` compliant? ;)

Comment (kwarunek) [reply src/asyncopenstackclient/client.py]: flake does not complain... it's ok according to pep8

File: README.rst (hunk 1 above)
Comment (giffels) [reply README.rst]: Fixed! My copy & paste mistake. ;-)

**Comment by evemorgen** [comment]
We're good to go then

**Comment by evemorgen** [comment]
It has been released under 0.7.0 tag

**Pull request merged**